wheel/src/api.rs

```diff
@@ -5,16 +5,34 @@ use clvmr::chia_dialect::ChiaDialect;
 use clvmr::cost::Cost;
 use clvmr::reduction::Response;
 use clvmr::run_program::run_program;
-use clvmr::serde::{node_from_bytes, serialized_length_from_bytes};
+use clvmr::serde::{node_from_bytes, parse_through_clvm_object, serialized_length_from_bytes};
 use clvmr::{LIMIT_HEAP, LIMIT_STACK, MEMPOOL_MODE, NO_UNKNOWN_OPS};
 use pyo3::prelude::*;
 use pyo3::wrap_pyfunction;
 
+struct ReadPyAny<'py>(&'py PyAny);
+
+impl<'py> std::io::Read for ReadPyAny<'py> {
```

Comment:
I assume this python interface doesn't support seeking forward. I think the work-around to just read the bytes that you want to skip should be moved here. `ReadPyAny` should implement `Seek` and do the trick.

(which still doesn't need a copy function btw, just to read and discard bytes)

Reply:
True, I could write my own loop to read bytes, but `copy` https://doc.rust-lang.org/std/io/fn.copy.html is parameterized by type, so the `Sink` writes should go away.

Reply:
you already did write your own loop reading bytes. you just also write them to another stream. what I mean is that if you're going to add this feature you should make sure to add tests for it as well. But given that we don't need it (the copy), it would make more sense to do none of them.

Reply:
Oh yeah, I suppose I could also pass a `Write` to `.skip_clvm_object`! That would be a lot more direct (and mean we don't have to call out to python).